examples/hybrid/driver.jl

```diff
@@ -119,10 +120,41 @@ end
 
 if config.parsed_args["check_conservation"]
     FT = Spaces.undertype(axes(sol.u[end].c.ρ))
-    @test sum(sol.u[1].c.ρ) ≈ sum(sol.u[end].c.ρ) rtol = 50 * eps(FT)
-    @test sum(sol.u[1].c.ρe_tot) +
-          (p.net_energy_flux_sfc[][] - p.net_energy_flux_toa[][]) ≈
-          sum(sol.u[end].c.ρe_tot) rtol = 100 * eps(FT)
+
+    # energy
+    energy_total = sum(sol.u[end].c.ρe_tot) - p.net_energy_flux_toa[][]
+    energy_atmos_change = sum(sol.u[end].c.ρe_tot) - sum(sol.u[1].c.ρe_tot)
+    sfc = p.atmos.surface_model
+    if sfc isa CA.PrognosticSurfaceTemperature
+        sfc_cρh = sfc.ρ_ocean * sfc.cp_ocean * sfc.depth_ocean
+        energy_total +=
+            horizontal_integral_at_boundary(sol.u[end].sfc.T .* sfc_cρh)
+        energy_surface_change =
+            horizontal_integral_at_boundary(
+                sol.u[end].sfc.T .- sol.u[1].sfc.T,
+            ) * sfc_cρh
+    else
+        energy_surface_change = 0
+    end
+    energy_radiation_input = -p.net_energy_flux_toa[][]
+    @test (energy_atmos_change + energy_surface_change) / energy_total ≈
+          energy_radiation_input / energy_total atol = sqrt(eps(FT))
+
+    if p.atmos.moisture_model isa DryModel
+        # density
+        @test sum(sol.u[1].c.ρ) ≈ sum(sol.u[end].c.ρ) rtol = 50 * eps(FT)
+    else
+        # water (some error due to not accumulating P at every substage)
+        water_total =
+            sum(sol.u[end].c.ρq_tot) .-
+            horizontal_integral_at_boundary(sol.u[end].sfc.P_minus_E)
+        water_atmos_change = sum(sol.u[end].c.ρq_tot) - sum(sol.u[1].c.ρq_tot)
+        water_surface_change = horizontal_integral_at_boundary(
```

Comment:
It might be more clear to call it something like `water_surface_input`?

Reply:
It's not really an input to the evolving (atmosphere-surface) system when the surface is prognostic, so I'd prefer the more general name, if that works for you?

Reply:
Sounds good!